server/service/src/stocktake/update.rs

```diff
@@ -339,16 +341,16 @@ fn generate_stock_in_out_or_update(
             batch: row.batch,
             pack_size: row.pack_size,
             expiry_date: row.expiry_date,
-            cost_price_per_pack: None,
-            sell_price_per_pack: None,
+            cost_price_per_pack: Some(cost_price_per_pack),
```

Comment:
Not sure why the cost and sell prices weren't being updated for stock out? If the stock line hasn't been zero'd then the cost and sell prices should still update?